### ISSUE TITLE
No module named lzma

### ISSUE THREAD

**Issue by pkMyt1** [jdidion/atropos#54]
I downloaded this and tried a local install using python3 setup.py install --user.  Now when I run it I get an error that I have been unable to trace.  The full error stack is

[dsimpson@localhost atropos]$ atropos
Traceback (most recent call last):
  File "/home/dsimpson/.local/bin/atropos", line 4, in <module>
    __import__('pkg_resources').run_script('atropos==0+unknown', 'atropos')
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 750, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1527, in run_script
    exec(code, namespace, namespace)
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/EGG-INFO/scripts/atropos", line 26, in <module>
    main()
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/EGG-INFO/scripts/atropos", line 23, in main
    sys.exit(execute_cli(args))
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/commands/__init__.py", line 189, in execute_cli
    print_subcommands()
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/commands/__init__.py", line 232, in print_subcommands
    command.get_help() for command in iter_commands())))
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/commands/__init__.py", line 232, in <genexpr>
    command.get_help() for command in iter_commands())))
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/commands/__init__.py", line 101, in get_help
    name=self.name, description=self.description.strip())
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/commands/__init__.py", line 95, in description
    return self.get_command_parser_class().description
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/commands/__init__.py", line 82, in get_command_parser_class
    mod = import_module(self.cli_module)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/commands/detect/__init__.py", line 8, in <module>
    from atropos.commands.base import (
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/commands/base.py", line 8, in <module>
    from atropos.adapters import AdapterCache
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/adapters/__init__.py", line 15, in <module>
    from atropos.io.seqio import ColorspaceSequence, FastaReader
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/io/__init__.py", line 8, in <module>
    from atropos.io.compression import get_file_opener
  File "/home/dsimpson/.local/lib/python3.5/site-packages/atropos-0+unknown-py3.5-linux-x86_64.egg/atropos/io/compression.py", line 6, in <module>
    import lzma
  File "/usr/lib/python3.5/lzma.py", line 26, in <module>
    from _lzma import *
ImportError: No module named '_lzma'

I am assuming I did something wrong here, any ideas?  Thank you.

**Comment by jdidion** [comment]
This is quite strange. Your python installation is missing the lzma library, which was added in python 3.3.  Are you using a binary installation of cpython, or did you build from source, or are you using something else (e.g. IronPython or PyPy)?

**Comment by pkMyt1** [comment]
The lzma module is there.  To make doubly certain I just reinstalled everything.  I get the error above from pip or setup install.  Running atropos from GitHub or after a build but no install I get

[dsimpson@localhost atropos]$ bin/atropos --help
Traceback (most recent call last):
  File "bin/atropos", line 6, in <module>
    from atropos import check_importability
ImportError: No module named atropos

I had it working two weeks ago but wanted a clean set of files so I downloaded a fresh copy.  I will chase this some more.
